docs/users/allowed-code.rst

```diff
@@ -0,0 +1,87 @@
+=============
+Code Allowed in WordPress/Largo Posts and Pages
```

Comment:
Added to the Largo Guide for Authors: https://largo.inn.org/guides/authors/post-edit-screen/embedding-interactives/